src/main/java/seedu/address/storage/JsonSerializableTaskHub.java

```diff
@@ -25,15 +26,18 @@ class JsonSerializableTaskHub {
 
     private final List<JsonAdaptedEmployee> employees = new ArrayList<>();
     private final List<JsonAdaptedProject> projects = new ArrayList<>();
+    private final List<JsonAdaptedTask> tasks = new ArrayList<>();
 
     /**
      * Constructs a {@code JsonSerializableTaskHub} with the given employees.
      */
     @JsonCreator
     public JsonSerializableTaskHub(@JsonProperty("employees") List<JsonAdaptedEmployee> employees,
-                                   @JsonProperty("projects") List<JsonAdaptedProject> projects) {
+                                   @JsonProperty("projects") List<JsonAdaptedProject> projects,
+                                   @JsonProperty("tasks") List<JsonAdaptedTask> tasks) {
```

Comment:
I believe the way you are doing it now is creating an extra section of the JSON that is just storing only tasks, on top of those same tasks being stored in each individual project. I tried deleting a task from that extra section and TaskHub can still runs and with that task still being stored in the same project(I did not delete the task that was being stored in the project). I'm not too sure myself if that extra section is needed, so maybe this is redundant?

Reply:
Originally, I was using the code for employees and projects to build the task model, hence `tasks` sort of exist in their own JSON list as well. It is true that they are now duplicated in their own JSON list and in the projects that they belong to, but if we were to implement something like `findT` or `listT`, keeping this might be worth it. But it's open to discussion! Thanks for noticing this.